VAMobile/src/components/MessageAlert.tsx

```diff
@@ -23,6 +28,16 @@ export type MessageAlertProps = {
 const MessageAlert: FC<MessageAlertProps> = ({ hasValidationError, saveDraftAttempted, scrollViewRef, focusOnError, errorList }) => {
   const theme = useTheme()
   const { t } = useTranslation(NAMESPACE.HEALTH)
+  const { replyTriageError } = useSelector<RootState, SecureMessagingState>((state) => state.secureMessaging)
```

Comment:
At first glance, I felt like this and the related code could be removed from of the component, and a prop could be used instead for deciding when to display the triage error to keep the component simpler and more general (because this is exclusive to the reply message screen). But looking at what the error does, it'd be cumbersome to handle the resets when navigating back to the secure messaging screen if this and related code wasn't in here (although it could be done with `useBeforeNavBackListener` in the reply message screen), so I think this is fine for now. This component could get cluttered if we get more message errors types in the future, so if that happens, we may want to look into refactoring stuff like this

Reply:
I'd personally love to delete MessageAlert all together. It feels like a lot of layered on complexity for very little removal of duplicative code.

Reply:
💯